docs/source/tracking.rst

```diff
@@ -187,19 +187,60 @@ common tasks:
 Running a Tracking Server
 -------------------------
 
-The MLflow tracking server launched via ``mlflow ui`` also hosts REST APIs for tracking runs,
+The MLflow tracking server launched via ``mlflow server`` also hosts REST APIs for tracking runs,
 writing data to the local filesystem. You can specify a tracking server URI
 via the ``MLFLOW_TRACKING_URI`` environment variable and MLflow's tracking APIs will automatically
 communicate with the tracking server at that URI to create/get run information, log metrics, etc.
 
-For example, to launch a run against a local tracking server, launch ``mlflow ui``, set
-``MLFLOW_TRACKING_URI`` to ``http://localhost:5000``, and run:
+When running the remote server, it is recommended that you provide the `--artifact-root` command
+line option. This option will cause clients to log their artifact output (e.g., models) to this
+location which is suitable for large data (such as an S3 bucket). If you do not provide this option,
+then clients will write artifacts to `their` local directories, which the server can't serve.
```

Comment:
You might want to use a normal FS if you've mounted it on the same path on the clients and server. I'd mention that since many environments do have some form of shared NFS. Basically the two options are shared filesystem (must be mounted in the same place everywhere) or S3.

Reply:
Updated the example to include shared NFS.